pkg/convert/python/python.go

```diff
@@ -309,7 +309,12 @@ func (c *PythonContext) generatePackage(pack Package, version string) config.Pac
 
 	c.Logger.Printf("[%s] Run time Deps %v", pack.Info.Name, pack.Dependencies)
 
-	pack.Dependencies = append(pack.Dependencies, "python-"+c.PythonVersion)
+	// The package is called python3, but versions are provided with python-3.xx
```

Comment:
Wolfi provides both the `python3` and `python-3` virtuals now, so this shouldn't be necessary.

Reply:
yAY!